docs/android-core-new/participants/events.mdx

```diff
@@ -1,192 +1,230 @@
 ---
-title: Events
-description: Event handling for participants.
+title: Participant Events
+description: >-
+  Dive into the details of handling participant events in your Android
+  application using Dyte's comprehensive documentation.
 sidebar_position: 3
 tags:
-  - web-core
+  - android-core
   - participants
   - self
 ---
 
-You can subscribe to events for all participants using
-`meeting.participants.on()` method. Here are the supported events:
+# All Participant Events
 
-### View mode change
+You can subscribe to events for all participants by implementing
+`DyteParticipantEventsListener` callback and then passing that object to
+`meeting.addParticipantEventsListener(dyteParticipantEventsListener)` method.
 
-Triggered when the View mode changes
+Here are the supported methods:
 
-```ts
-meeting.participants.on(
-  'viewModeChanged',
-  ({ viewMode, currentPage, pageCount }) => {
-    console.log('view mode changed', viewMode);
-  }
-);
+## Participant joined
+
+Triggers an event when any participant joins the meeting.
+
+```kotlin
+    private val participantEventsListener = object : DyteParticipantEventsListener {
+        override fun onParticipantJoin(participant: DyteMeetingParticipant) {
+            // your code here to handle new participant
+        }
+    }
 ```
 
-### Page change
+## Participant left
 
-```ts
-meeting.participants.on(
-  'pageChanged',
-  ({ viewMode, currentPage, pageCount }) => {
-    console.log('page changed', currentPage);
-  }
-);
+Triggers an event when any participant leaves the meeting.
+
+```kotlin
+    private val participantEventsListener = object : DyteParticipantEventsListener {
+        override fun onParticipantLeave(participant: DyteMeetingParticipant) {
+            // your code here to handle participant left from meeting
+        }
+    }
 ```
 
-### Active speaker
+## Participant update
```

Comment:
```suggestion
## Participants update
```
This callback is given if there is a change in any of the properties of the meeting.participants object. Including changes in any individual participant.